core/src/com/unciv/logic/automation/Automation.kt

```diff
@@ -352,10 +352,12 @@ object Automation {
         val stats = tile.getTileStats(null, civInfo)
         var rank = rankStatsValue(stats, civInfo)
         if (tile.improvement == null) rank += 0.5f // improvement potential!
+        if (tile.isPillaged()) rank += 0.6f
         if (tile.hasViewableResource(civInfo)) {
             val resource = tile.tileResource
             if (resource.resourceType != ResourceType.Bonus) rank += 1f // for usage
             if (tile.improvement == null) rank += 1f // improvement potential - resources give lots when improved!
+            if (tile.isPillaged()) rank += 1.1f // even better, repair is faster
```

Comment:
So this adds the bonus twice, for a total of 1.7f?

Reply:
Yes, similar to the tile.improvement getting 0.5f for needing an improvement, then +1f for if ViewableResource for a 1.5 total